service/src/test/java/skills/intTests/utils/SkillsService.groovy

```diff
@@ -789,6 +789,14 @@ class SkillsService {
         wsHelper.apiGet(url)
     }
 
+    def getSubjectSummaryForUser(String projId, String subjectId, String userId, int version = -1) {
```

Comment:
looks like this endpoint can already be called via ``skills.intTests.utils.SkillsService#getSkillSummary(java.lang.String, java.lang.String, java.lang.String, int, boolean)``